docs/framework/install/net-framework-not-found.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Troubleshooting 'This application could not be started'
+description: Learn what to do if you see a 'This application could not be started' dialog box.
+author: rpetrusha
+ms.author: ronpet
+ms.date: 09/05/2019
+---
+# Troubleshooting a 'This application could not be started' error message
+
+Applications that are developed for the .NET Framework typically require that a specific version of the .NET Framework be installed on your system. In some cases, you may attempt to run an application without either an installed version or the expected version of the .NET Framework present. This often produces an error dialog box like the following:
+
+![This application could not be started](media/app-could-not-be-started.png)
+
+This typically indicates one of the following conditions:
+
+- A .NET Framework installation on your system has become corrupted.
+
+- The version of the .NET Framework needed by your application cannot be detected.
+
+To address this issue so that you can run your application, do the following:
+
+1. Download the [.NET Framework Repair Tool (NetFxRepairTool.exe)](https://www.microsoft.com/en-us/download/details.aspx?id=30135). The tool runs automatically when the download completes.
+
+1. If the .NET Framework Repair Tool recommends any additional action, such as those shown in the following figure, select **Next**.
+
+   ![Recommended changes](media/repair-tool-recommended-changes.png)
```

Comment:
these screenshots look a bit trimmed. not sure if we can retake them to have the borders?

Reply:
I think that they're as good as we can get -- the problem is that the dialog box has no borders. (Although I'll try it on another machine to make sure the UI is the same.)

Reply:
Though I've replaced one of them.